docs/dev/clusteroperator.md

```diff
@@ -64,9 +64,15 @@ Refer [the godocs](https://godoc.org/github.com/openshift/api/config/v1#ClusterS
 
 In general, ClusterOperators should contain at least three core conditions:
 
-* `Progressing` which is true if the controller has detected the state is invalid and is working towards it (or if the controller has never reconciled before)
-* `Available` should be true if the controller has successfully reached a working status and no known errors are blocking progress.
-* `Failing` should be true if the controller is blocked from reaching the desired state.
+* `Progressing` must be true if the operator is actually making change to the operand.
```

Comment:
We should clarify what to do when we don't have an operator per se.  I'll add that in as a follow up PR.